source/releasenotes/2024-08-23-emergency-ticket.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Given the site dashboard support context will still say "On-Call" ([see related convo here](https://github.com/pantheon-systems/documentation/pull/9191/files#r1729305038)) I think the release note copy needs to be reconsidered. Do we need a release note announcing copy changes?

Reply:
The goal is to remove all references to "on-call" everywhere. We should document both the Phoebe and Hermes experiences, but even Hermes should be getting updated to avoid "on-call" language, right?

Reply:
> but even Hermes should be getting updated to avoid "on-call" language, right?

@stephpawlowski can you confirm?

Reply:
per slack convo, i'm deleting the release note from this PR, but i have it preserved on my local and will add it back in a follow-up docs PR which is to be released alongside hermes copy changes